std/hash/wyhash.zig

```diff
@@ -10,7 +10,8 @@ const primes = [_]u64{
 };
 
 fn read_bytes(comptime bytes: u8, data: []const u8) u64 {
-    return mem.readVarInt(u64, data[0..bytes], .Little);
+    const T = @IntType(false, 8 * bytes);
+    return mem.readIntSliceLittle(T, data[0..bytes]);
```

Comment:
This was the main speed improvement. `mem.readVarInt` is very slow and I've done this change before. I think this api could be improved or at least documented a bit better. We want to simply reinterpret bits and byteswap if needed, which this function does.